gradio/blocks.py

```diff
@@ -983,8 +1002,9 @@ def load(
         api_key: Optional[str] = None,
         alias: Optional[str] = None,
         _js: Optional[str] = None,
+        every: None | int = None,
```

Comment:
Unrelated to your PR, this event trigger is missing many arguments, such as batch. It would be good to add if possible (or we can create a separate issue for it)

Reply:
I'll do that in a separate PR! Thank you for pointing this out!